src/scenes/Authentication/Register/RegisterForm.tsx

```diff
@@ -38,8 +38,10 @@ const useStyles = makeStyles(({ spacing }) => ({
 }));
 
 interface RegisterInput {
```

Comment:
This interface could be pulled from the schema or at least extend from it since we need to add `confirmPassword`.

Reply:
Pulled from schema